src/Google/Ads/GoogleAds/Util/Testing/testdata/test_cases.json

```diff
@@ -108,5 +108,46 @@
       }
     },
     "expectedMask": "foo.bar.nestedWrapper"
+  }, {
```

Comment:
Could you add tests when things do not change?

I am thinking:

- Both set with same value
- Both unset

Reply:
With the test cases as JSON, 
* Both unset (I guess you mean the presence field) would be similar to when both original and modified resources are empty. We cannot set the presence field to null, as it requires to have an int type.
* Both set with same value --> This is now added.

Reply:
LGTM